serverless/pytorch/ultralytics/yolov5/nuclio/main.py

```diff
@@ -0,0 +1,41 @@
+import json
+import base64
+from PIL import Image
+import io
+import yaml
```

Comment:
The import is unused (pylint complains about it). Could you please remove it?

Reply:
the yaml import was the unused one, I've removed it!